--- cinelerra-cv/plugins/shapewipe/shapewipe.C~	2010-01-09 15:12:21.000000000 +0100
+++ cinelerra-cv/plugins/shapewipe/shapewipe.C	2010-02-16 15:03:37.044711032 +0100
@@ -386,7 +386,7 @@
 	}
 
 	png_ptr = png_create_read_struct(PNG_LIBPNG_VER_STRING,
-		png_voidp_NULL, png_error_ptr_NULL, png_error_ptr_NULL);
+		NULL, NULL, NULL);
 
 	if (!png_ptr)
 	{
@@ -399,14 +399,14 @@
 	info_ptr = png_create_info_struct(png_ptr);
 	if (!info_ptr)
 	{
-		png_destroy_read_struct(&png_ptr, png_infopp_NULL, png_infopp_NULL);
+		png_destroy_read_struct(&png_ptr, NULL, NULL);
 		return 1;
 	}
 
 	end_info = png_create_info_struct(png_ptr);
 	if (!end_info)
 	{
-		png_destroy_read_struct(&png_ptr, &info_ptr, png_infopp_NULL);
+		png_destroy_read_struct(&png_ptr, &info_ptr, NULL);
 		return 1;
 	}
 
