--- cinelerra-cv-2.1/configure.in.org	2006-11-26 03:33:50.056844000 +0100
+++ cinelerra-cv-2.1/configure.in	2006-11-26 03:33:55.053156250 +0100
@@ -99,14 +99,6 @@
 	LIBS=`echo "$LIBS" | sed -e s/-lasound//g`
 fi
 
-if test "x$enable_esd" = "xyes"; then
-	AM_PATH_ESD(,[esound=yes])
-	if test "x$esound" = xyes; then
-		SOUND_CFLAGS="$ESD_CFLAGS -DHAVE_ESOUND $SOUND_CFLAGS"
-		SOUND_LDFLAGS="$ESD_LIBS $SOUND_LDFLAGS"
-		enable_esd=ok
-	fi
-fi
 AC_SUBST(SOUND_CFLAGS)
 AC_SUBST(SOUND_LDFLAGS)
 ############### END OF SOUND
--- cinelerra-cv-2.1/libmpeg3/video/Makefile.am	2008-08-23 14:53:19.959321239 +0000
+++ cinelerra-cv-2.1/libmpeg3/video/Makefile.am.org	2008-08-23 14:53:15.609327436 +0000
@@ -11,6 +11,7 @@
 
 if USEMMX
 libmpeg3_video_la_SOURCES += mmxidct.S reconmmx.s
+AM_LIBTOOLFLAGS=--tag=CC
 else
 libmpeg3_video_la_SOURCES +=
 endif
--- cinelerra-cv-2.1/mpeg2enc/Makefile.am~	2008-08-23 16:56:56.992840914 +0200
+++ cinelerra-cv-2.1/mpeg2enc/Makefile.am	2008-08-23 17:00:05.433238449 +0200
@@ -17,6 +17,7 @@
 libmpeg2enc_la_SOURCES +=     fdct_mmx.s fdctdata.c idct_mmx.s idctdata.c \
     quant_mmx.s quantize_x86.c predict_mmx.s \
     predcomp_mmxe.s predcomp_mmx.s
+AM_LIBTOOLFLAGS=--tag=CC
 endif
 
 noinst_HEADERS = attributes.h \
--- cv/configure.in.org	2008-10-27 10:15:06.091895546 +0100
+++ cv/configure.in	2008-10-27 10:15:21.304615704 +0100
@@ -356,12 +356,7 @@
      dnl if not, that means that libswscale is compiled in
      
      AC_MSG_CHECKING(for ffmpeg swscale support)
-     saved_LIBS="$LIBS"
-     LIBS="$saved_LIBS $FFMPEG_TEMP_LIBS"
-     AC_TRY_LINK([#include <libavcodec/avcodec.h>],
-		 [img_convert(0, 0, 0,0,0,0)],
-		 enable_ffmpeg_swscale=no,enable_ffmpeg_swscale=yes)
-     LIBS="$saved_LIBS"
+     enable_ffmpeg_swscale=yes
      AC_MSG_RESULT($enable_ffmpeg_swscale)
      if test x"$enable_ffmpeg_swscale" = xyes; then
 	     dnl AC_DEFINE(HAVE_SWSCALER)
--- cv/cinelerra/ffmpeg.C.org	2008-10-27 10:26:46.890782761 +0100
+++ cv/cinelerra/ffmpeg.C	2008-10-27 10:27:19.937445860 +0100
@@ -2,6 +2,7 @@
 
 #ifdef HAVE_SWSCALER
 extern "C" {
+#include <avcodec.h>
 #include <swscale.h>
 }
 #endif
--- cinelerra-cv/plugins/svg/svg.C~	2010-01-09 15:12:21.000000000 +0100
+++ cinelerra-cv/plugins/svg/svg.C	2010-06-03 17:56:46.125079418 +0200
@@ -29,6 +29,8 @@
 #include <string.h>
 #include <errno.h>
 #include <sys/mman.h>
+#include <sys/types.h>
+#include <sys/stat.h>
 
 
 #include <libintl.h>

--- cinelerra-cv/guicast/Makefile.am~	2012-11-18 13:43:23.212827961 +0100
+++ cinelerra-cv/guicast/Makefile.am	2012-11-18 13:55:19.386136040 +0100
@@ -1,7 +1,7 @@
 lib_LTLIBRARIES = libguicast.la
 noinst_PROGRAMS=bootstrap
 
-libguicast_la_LIBADD = $(OPENGL_LIBS) -lXxf86vm -lXv -lXext -lX11 $(X_EXTRA_LIBS)
+libguicast_la_LIBADD = $(OPENGL_LIBS) -lXxf86vm -lXv -lXext -lX11 -lpng ../quicktime/libquicktimehv.la $(X_EXTRA_LIBS)
 libguicast_la_LDFLAGS = $(X_LIBS) -version-info 1:0:0 
 libguicast_la_SOURCES = \
 	bcbar.C \
--- cinelerra-cv/cinelerra/vdevicebuz.C~	2013-09-21 22:36:30.000000000 +0200
+++ cinelerra-cv/cinelerra/vdevicebuz.C	2013-11-07 09:02:07.102426371 +0100
@@ -44,6 +44,7 @@
 
 #include <errno.h>
 #include <stdint.h>
+#include <string.h>
 #include <linux/kernel.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
--- cinelerra-cv/cinelerra/videodevice.C~	2013-09-21 22:36:30.000000000 +0200
+++ cinelerra-cv/cinelerra/videodevice.C	2013-11-07 09:02:50.862426055 +0100
@@ -51,6 +51,7 @@
 #include "videowindowgui.h"
 #include "vframe.h"
 
+#include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
 
