--- cinelerra-cv-2.1/configure.in.org	2006-11-26 03:33:50.056844000 +0100
+++ cinelerra-cv-2.1/configure.in	2006-11-26 03:33:55.053156250 +0100
@@ -7,7 +7,11 @@
 
 AM_INIT_AUTOMAKE
 AC_CONFIG_HEADER(config.h)
-AM_GNU_GETTEXT([external])
+
+AC_PROG_CC
+AC_GNU_SOURCE
+
+AM_GNU_GETTEXT
 AM_GNU_GETTEXT_VERSION(0.12.1)
 AM_PROG_AS
 
@@ -99,14 +99,6 @@
 	LIBS=`echo "$LIBS" | sed -e s/-lasound//g`
 fi
 
-if test "x$enable_esd" = "xyes"; then
-	AM_PATH_ESD(,[esound=yes])
-	if test "x$esound" = xyes; then
-		SOUND_CFLAGS="$ESD_CFLAGS -DHAVE_ESOUND $SOUND_CFLAGS"
-		SOUND_LDFLAGS="$ESD_LIBS $SOUND_LDFLAGS"
-		enable_esd=ok
-	fi
-fi
 AC_SUBST(SOUND_CFLAGS)
 AC_SUBST(SOUND_LDFLAGS)
 ############### END OF SOUND
--- cinelerra-cv-2.1/cinelerra/Makefile.am.org	2006-11-26 01:42:58.101122750 +0100
+++ cinelerra-cv-2.1/cinelerra/Makefile.am	2006-11-26 01:43:09.741850250 +0100
@@ -19,26 +19,26 @@
 
 if BUILDINFO_GIT
 GITVERSION=$(shell if [ -d $(top_srcdir)/.git ]; then cd $(top_srcdir); git rev-parse HEAD; else echo "unknown"; fi)
-	echo '#define REPOMAINTXT " GIT::$(GITVERSION) $(BUILDINFO_NONRC_S) (C) 2006 Heroine Virtual Ltd.\n@FFMPEG_EXTERNALTEXT@\nCompiled on $(BUILDDATE)"' >> versioninfo.h
+	echo '#define REPOMAINTXT " GIT::$(GITVERSION) $(BUILDINFO_NONRC_S) (C) 2006 Heroine Virtual Ltd. @FFMPEG_EXTERNALTEXT@ Compiled on $(BUILDDATE)"' >> versioninfo.h
 	echo '#define REPOABOUTPREFTXT "Git SHA-1: $(GITVERSION)"' >> versioninfo.h
 else
 
 if BUILDINFO_SVN
 SVNVERSION=$(shell if [ -d $(top_srcdir)/.svn ]; then svnversion $(top_srcdir); else echo "unknown"; fi)
-	echo '#define REPOMAINTXT      " SVN $(SVNVERSION) $(BUILDINFO_NONRC_S) (C) 2006 Heroine Virtual Ltd.\n@FFMPEG_EXTERNALTEXT@\nCompiled on $(BUILDDATE)"' >> versioninfo.h
+	echo '#define REPOMAINTXT      " SVN $(SVNVERSION) $(BUILDINFO_NONRC_S) (C) 2006 Heroine Virtual Ltd. @FFMPEG_EXTERNALTEXT@ Compiled on $(BUILDDATE)"' >> versioninfo.h
 	echo '#define REPOABOUTPREFTXT "SVN Version: $(SVNVERSION) $(BUILDINFO_NONRC_S)"' >> versioninfo.h
 
 else
 if BUILDINFO_CUST
 
-	echo '#define REPOMAINTXT      " $(BUILDINFO_CUSTOM_TAG) (C) 2006 Heroine Virtual Ltd.\n@FFMPEG_EXTERNALTEXT@\nCompiled on $(BUILDDATE)"' >> versioninfo.h
+	echo '#define REPOMAINTXT      " $(BUILDINFO_CUSTOM_TAG) (C) 2006 Heroine Virtual Ltd. @FFMPEG_EXTERNALTEXT@ Compiled on $(BUILDDATE)"' >> versioninfo.h
 	echo '#define REPOABOUTPREFTXT "$(BUILDINFO_CUSTOM_TAG)"' >> versioninfo.h
 
 
 else # (no repository)
 
 
-	echo '#define REPOMAINTXT      "(C) 2006 Heroine Virtual Ltd.\nCompiled on $(BUILDDATE)"' >> versioninfo.h
+	echo '#define REPOMAINTXT      "(C) 2006 Heroine Virtual Ltd. Compiled on $(BUILDDATE)"' >> versioninfo.h
 	echo '#define REPOABOUTPREFTXT ""' >> versioninfo.h
 endif
 endif
--- cinelerra-cv-2.1/libmpeg3/video/Makefile.am	2008-08-23 14:53:19.959321239 +0000
+++ cinelerra-cv-2.1/libmpeg3/video/Makefile.am.org	2008-08-23 14:53:15.609327436 +0000
@@ -11,6 +11,7 @@
 
 if USEMMX
 libmpeg3_video_la_SOURCES += mmxidct.S reconmmx.s
+AM_LIBTOOLFLAGS=--tag=CC
 else
 libmpeg3_video_la_SOURCES +=
 endif
--- cinelerra-cv-2.1/mpeg2enc/Makefile.am~	2008-08-23 16:56:56.992840914 +0200
+++ cinelerra-cv-2.1/mpeg2enc/Makefile.am	2008-08-23 17:00:05.433238449 +0200
@@ -17,6 +17,7 @@
 libmpeg2enc_la_SOURCES +=     fdct_mmx.s fdctdata.c idct_mmx.s idctdata.c \
     quant_mmx.s quantize_x86.c predict_mmx.s \
     predcomp_mmxe.s predcomp_mmx.s
+AM_LIBTOOLFLAGS=--tag=CC
 endif
 
 noinst_HEADERS = attributes.h \
--- cv/configure.in.org	2008-10-27 10:15:06.091895546 +0100
+++ cv/configure.in	2008-10-27 10:15:21.304615704 +0100
@@ -356,12 +356,7 @@
      dnl if not, that means that libswscale is compiled in
      
      AC_MSG_CHECKING(for ffmpeg swscale support)
-     saved_LIBS="$LIBS"
-     LIBS="$saved_LIBS $FFMPEG_TEMP_LIBS"
-     AC_TRY_LINK([#include <libavcodec/avcodec.h>],
-		 [img_convert(0, 0, 0,0,0,0)],
-		 enable_ffmpeg_swscale=no,enable_ffmpeg_swscale=yes)
-     LIBS="$saved_LIBS"
+     enable_ffmpeg_swscale=yes
      AC_MSG_RESULT($enable_ffmpeg_swscale)
      if test x"$enable_ffmpeg_swscale" = xyes; then
 	     dnl AC_DEFINE(HAVE_SWSCALER)
--- cv/cinelerra/ffmpeg.C.org	2008-10-27 10:26:46.890782761 +0100
+++ cv/cinelerra/ffmpeg.C	2008-10-27 10:27:19.937445860 +0100
@@ -2,6 +2,7 @@
 
 #ifdef HAVE_SWSCALER
 extern "C" {
+#include <avcodec.h>
 #include <swscale.h>
 }
 #endif
--- cinelerra-cv/cinelerra/edits.C~	2010-01-09 15:12:19.000000000 +0100
+++ cinelerra-cv/cinelerra/edits.C	2010-04-26 09:13:43.381588613 +0200
@@ -47,7 +47,6 @@
 	this->edl = edl;
 	this->track = track;
 
-	List<Edit>::List<Edit>();
 	default_edit->edl = edl;
 	default_edit->track = track;
 	default_edit->startproject = 0;
